docs/tutorials/my-first-custom-action.rst

```diff
@@ -0,0 +1,255 @@
+
+Okay! We find ``on_pod_create`` and ``on_event_create``
+
+We’ll use ``on_event_create``, because in this case ``on_pod_create`` is not even called because the pod is not created.
+
+Writing the action
+----
+
+Now we need to write code that checks this event and reports it. To find out the correct event class that matches our trigger ``on_event_create``. please take a look at :ref:`Event Hierarchy`.
+
+Okay! We find out it’s ``EventEvent``!
+
+So we need to get the information, check for the scenario, and then report it (for more information about reporting it see :ref:`Findings API`)
+
+Let’s name our action ``report_scheduling_failure``, and write everything in a python file:
+
+.. code-block:: python
+
+    from robusta.api import *
+
+    @action
+    def report_scheduling_failure(event: EventEvent):
+        actual_event = event.get_event()
+
+        print(f"This print will be shown in the robusta logs={actual_event}")
+
+        if actual_event.type.casefold() == f'Warning'.casefold() and \
+            actual_event.reason.casefold() == f'FailedScheduling'.casefold() and \
+            actual_event.involvedObject.kind.casefold() == f'Pod'.casefold():
+            _report_failed_scheduling(event, actual_event.involvedObject.name, actual_event.message)
+
+    def _report_failed_scheduling(event: EventEvent, pod_name: str, message: str):
+        # this is how you send data to slack or other destinations
+        event.add_enrichment([
+            MarkdownBlock(f"Failed to schedule a pod named '{pod_name}', error: {message}"),
+        ])
+
+Before we proceed, we need to enable local playbook repositories in Robusta.
+
+Follow this quick guide to learn how to package your python file for Robusta: :ref:`Custom playbook repositories`
+
+Use this useful debugging commands to make sure your action ( ``report_scheduling_failure``) is loaded:
+
+.. code-block:: bash
+
+    robusta logs # get robusta logs, see errors.
+    robusta playbooks list-dirs  # get see if you custom action was loaded
+
+Let’s push the new action to Robusta, and then test it by triggering the action manually immediately.
+
+.. code-block:: bash
+
+    robusta playbooks push <PATH_TO_LOCAL_PLAYBOOK_FOLDER>
+    robusta playbooks trigger report_scheduling_failure name=robusta-runner-8cd69f7cb-g5bkb namespace=default seconds=5
+
+Check our slack channel, and:
+
+.. image:: /images/example_report_scheduling_failure.png
+
+Connection the trigger to the action - a Playbook is born!
+-------------------------------------
+
+We need to add a custom playbook that this action it in the generated_values.yaml.
+
+.. code-block:: yaml
+
+    globalConfig:
+      signing_key: XXXX
+      account_id: XXXX
+    sinksConfig:
+    - slack_sink:
+        name: main_slack_sink
+        slack_channel: '#my-slack-channel'
+        api_key: XXXXX
+    - robusta_sink:
+        name: robusta_ui_sink
+        token: XXXXXX  # generated with `robusta gen-config`
+    clusterName: kind-kind
```

Comment:
This is the first time we talk about kind. 
Is another clusterName is better here?